service/scripts/dbSetup.ts

```diff
@@ -10,6 +10,123 @@ dotenv.config();
 
 const DB_URL = process.env.DATABASE_URL || 'mongodb://localhost:27017/measure-repository';
 const COLLECTION_NAMES = ['Measure', 'Library'];
+const ECQM_CONTENT_PATH = 'ecqm-content-qicore-2024/bundles/measure';
+const MEASURES_PATH = path.join(ECQM_CONTENT_PATH);
+
+/**
+ * Gets the paths for the bundles in ecqm-content-qicore-2024
+ */
+function getBundlePaths(): string[] {
+  const filePaths: string[] = [];
+
+  fs.readdirSync(MEASURES_PATH, { withFileTypes: true }).forEach(ent => {
+    // if this item is a directory, look for .json files under it
+    if (ent.isDirectory()) {
+      fs.readdirSync(path.join(MEASURES_PATH, ent.name), { withFileTypes: true }).forEach(subEnt => {
+        if (!subEnt.isDirectory() && subEnt.name.endsWith('.json')) {
+          filePaths.push(path.join(MEASURES_PATH, ent.name, subEnt.name));
+        }
+      });
+    }
+  });
+  return filePaths;
+}
+
+/**
+ * Changes the incorrect ecqi.healthit.gov references on the Libraries in
+ * ecqm-content-qicore-2024 to the correct madie.cms.gov references
+ */
+async function fixAndPutLibraries(bundle: fhir4.Bundle, url: string) {
+  const libraries: fhir4.Library[] = bundle.entry
+    ?.filter(entry => entry.resource?.resourceType === 'Library')
+    .map(entry => entry.resource as fhir4.Library) as fhir4.Library[];
+
+  for (const library of libraries) {
+    console.log(`  Library ${library.id}`);
+    if (library.relatedArtifact) {
+      for (let index = 0; index < library.relatedArtifact.length; index++) {
+        const ra = library.relatedArtifact[index];
+        // if a related artifact is using a ecqi.healthit.gov reference it needs to be
+        // changed to a madie.cms.gov reference to match the urls the resources use
+        if (
+          (ra.type === 'depends-on' || ra.type === 'composed-of') &&
+          ra.resource?.startsWith('http://ecqi.healthit.gov/ecqms/Library')
+        ) {
+          const newRef = ra.resource.replace(
+            'http://ecqi.healthit.gov/ecqms/Library',
```

Comment:
```suggestion
            'http://ecqi.healthit.gov/ecqms/Library/',
```

These need to match properly to not end up with a `//` in the new url that ends up making a broken reference for the MRS.